youtube_dl/extractor/canvas.py

```diff
@@ -7,6 +7,8 @@
 
 
 class CanvasIE(InfoExtractor):
+    IE_NAME = 'VRT'
+    IE_DESC = 'Een.be and Canvas.be'    
```

Comment:
Use consistent naming and not capitalized here and non capitalized in different places.